makefiles/Makefile.docs-csfle-merge

```diff
@@ -14,8 +14,4 @@ REPO_DIR=$(shell pwd)
 SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
 SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
 
-include ~/shared.mk
-
-
-
-
+include ~/shared.mk
```

Comment:
Do we need a newline at the end of a makefile?

Reply:
oh, funky! It's probably fine, but I'd rather have one. Good catch!